railties/test/application/configuration_test.rb

```diff
@@ -1893,6 +1893,26 @@ def index
       assert_equal Logger::DEBUG, Rails.logger.level
     end
 
+    test "config.logger when logger is already a Broadcast Logger" do
+      logger = ActiveSupport::BroadcastLogger.new
+
+      make_basic_app do |application|
+        application.config.logger = logger
+      end
+      assert_same(logger, Rails.logger)
+    end
+
+    test "config.logger when logger is not a Broadcast Logger" do
+      logger = Logger.new(STDOUT)
+
+      make_basic_app do |application|
+        application.config.logger = logger
+      end
+
+      assert_instance_of(ActiveSupport::BroadcastLogger, Rails.logger)
```

Comment:
@Edouard-chin Is BroadcastLogger the default logger?

Reply:
Yes that's correct, the BroadcastLogger is the default logger class now.